server/neptune/gateway/event/check_run_handler.go

```diff
@@ -55,20 +66,34 @@ func (h *CheckRunHandler) Handle(ctx context.Context, event CheckRun) error {
 		h.Logger.DebugContext(ctx, "Ignoring non-atlantis checks event")
 		return nil
 	}
+	matches := checkRunRegex.FindStringSubmatch(event.Name)
+	if len(matches) != 2 {
+		return fmt.Errorf("unable to determine root name")
+	}
```

Comment:
Will this error out for PR shas that have `atlantis/plan` on it?

Should we instead filter for `atlantis/deploy` above?

Reply:
The checkRunRegex already filters for `atlantis/deploy` for all types of check run events (unlock/plan/rerun). For re-requested events, we also filter out PR based events by confirming the check run event came from the default branch.